### DISCUSSION TITLE
Is there any settings for timezone?

### DISCUSSION THREAD

**Discussion by MattiasLindahl** [nathanmarlor/foxess_em#129] [Q&A]
I live in Sweden and we have just installed a Fox H3 inverter and 20kWh battery.
When installing your energy managemeng integration I get a feeling that you have coded for AU time?

For example Next Dawn time is not showing our expected dawn here in sweden but tells me now 21 hours. Here the local time is now 12:23.

/Mattias

**Comment by nathanmarlor** [comment]
Thanks Mattias.

I haven't hard coded any timezones, it should be using the timezone set by the OS. If you're running in a container it's worth checking you're passing in the TZ environment variable, otherwise it'll default to UTC?

**Reply by MattiasLindahl** [reply]
Ok, I will check it tomorrow when a day has passed. I´m running HA OS so I don´t think it should be an issue with UTC time. But I will check a bit more :)

thanks!

**Comment by nathanmarlor** [comment]
@MattiasLindahl just checking in - did the issue with time resolve itself? I've just release 1.6.0b2 which has a fix in for incorrect dawn calculations.
